BraveShared/CertificatePinning.swift

```diff
@@ -141,16 +157,33 @@ public class PinningCertificateEvaluator: NSObject, URLSessionDelegate {
         }
     }
     
+    private func fatalErrorInDebugModeIfPinningFailed() {
+        if !AppConstants.buildChannel.isPublic {
+            assertionFailure("An SSL Pinning error has occurred")
+        }
+    }
+    
     public struct PinningOptions: OptionSet {
         public let rawValue: Int
         
         public init(rawValue: Int) {
             self.rawValue = rawValue
         }
         
+        /// System's default pinning policies
         public static let `default` = PinningOptions(rawValue: 1 << 0)
+        
+        /// Host Validation
         public static let validateHost = PinningOptions(rawValue: 1 << 1)
-        public static let allowSelfSigned = PinningOptions(rawValue: 1 << 2)
-        public static let all: PinningOptions = [.default, .validateHost, .allowSelfSigned]
+        
+        /// Anchor ONLY the specified trusts passed to `PinningCertificateEvaluator.evaluate(trust:host:)`
+        public static let anchorSpecificTrustsOnly = PinningOptions(rawValue: 1 << 2)
+        
+        /// Anchor the specified trusts passed to `PinningCertificateEvaluator.evaluate(trust:host:)`
+        /// Also anchors the default trusts of the System
+        public static let anchorSpecificAndSystemTrusts = PinningOptions(rawValue: 1 << 3)
+        
+        /// Default pinning policies + Host Validation + Anchor the specified trusts + Anchor system trusts
```

Comment:
Are the default pinning policies the ones that Apple ships?

Reply:
Yes. The default certificates are the ones from the system. IE: "Built-In anchor"
![image](https://user-images.githubusercontent.com/1530031/135866791-9c744d6f-4a44-41b1-a29a-d6eeb6febb8b.png)